### PR TITLE
Make sure flags are correctly updated when setting renddesc

### LINKED ISSUE #2413: Portrait videos (9:16) videos are not rendering properly
When I render portrait video in synfig, the video comes out squeezed. Like this:

https://user-images.githubusercontent.com/50204208/141672708-e897ac69-8782-45af-a773-1db3b48667ff.mp4

**Expected behavior**
It should come out like this:

https://user-images.githubusercontent.com/50204208/141672588-9e827818-a937-44fb-b2b5-8a58c3de0ad0.mp4

**System information:**
 - OS: Windows 10 64bit 21h2

 - Commit ID: SynfigStudio-1.5.1-2021.10.21-win64-2cb6c

**Additional comments**
Stable version doesn't have this problem.

### PR DESCRIPTION
**Pull request by FirasH**
Fixes #2413

### CODE REVIEW

File: synfig-studio/src/gui/renddesc.cpp [hunk 1]
```diff
@@ -233,12 +233,13 @@ Widget_RendDesc::~Widget_RendDesc()
 
 void Widget_RendDesc::set_rend_desc(const synfig::RendDesc &rend_desc)
 {
-	if(update_lock)return;
+	if(update_lock)
+		return;
+
```
Comment (rodolforg) [inline synfig-studio/src/gui/renddesc.cpp:238]: Please do those esthetic changes in a separate commit. =)
And rebase this PR

**Review by rodolforg — CHANGES_REQUESTED** [review]
And please rebase this PR

**Review by rodolforg — APPROVED** [review]
You could let those changes here, but in other commit :)

Approved, anyway. It does fix the issue

**Comment by ice0** [comment]
Merged. Thank you!

**Comment by morevnaproject** [comment]
Backported to 1.4.3 ^__^

**Pull request merged**